python/export_util.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+import mgp
+import json as js
+
+
+@mgp.read_proc
+def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
+    """
+    Procedure to export the whole database as JSON to a local file.
+
+    Parameters
+    ----------
+    path : str
+        Path to the file where JSON will be saved.
+    """
+    nodes = list()
+    relationships = list()
+    graph = list()
+
+    for vertex in ctx.graph.vertices:
+        labels = [label.name for label in vertex.labels]
+        properties = dict()
+
+        for key in vertex.properties.keys():
+            properties[key] = vertex.properties.get(key)
+
+        node = {
+            "id": vertex.id,
```

Comment:
These seem like a constant, please write them like this example: https://github.com/memgraph/mage/blob/main/python/mage/graph_coloring_module/parameters.py

Reply:
Currently, I have issues with this kind of enums, due to the sort_key=True when dumping to JSON. I'll check what I can do. Meanwhile, I have decided to create dataclasses for Node and Relationship, it looks nice. We can discuss it on today's sync :D

Reply:
This is solved with value :)